### ISSUE TITLE
how do you perform AdaBN on the convergent model, update all bn layers or just some bn layers

### ISSUE THREAD

**Issue by trantorrepository** [sciencefans/trojans-face-recognizer#7]

**Comment by jihaonew** [comment]
1. Set model to train mode.
2. We forward testset via the mini-batch without the (optimizer.step()) operation and only the mean and variance in all bn layers will be updated.
